fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/DefaultIdentifierTranslator.java

```diff
@@ -139,4 +139,9 @@ public String asString(final Resource subject) {
         return absPath;
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
Not supporting `hashCode()` seems like an a sign of a more fundamental issue.
But any ways, the related Sonar issue seems to be gone: http://sonar.fcrepo.org/drilldown/issues/1?&rule=findbugs%3AEQ_COMPARETO_USE_OBJECT_EQUALS&rule_sev=MAJOR&severity=MAJOR